src/main/java/com/microsoft/sqlserver/jdbc/SQLServerConnection.java

```diff
@@ -6224,7 +6224,7 @@ final boolean complete(LogonCommand logonCommand, TDSReader tdsReader) throws SQ
         tdsWriter.writeShort((short) 0);
         tdsWriter.writeShort((short) 0);
 
-        if (tdsVersion >= TDS.VER_YUKON) {
+        if (tdsVersion >= TDS.VER_YUKON || tdsVersion == TDS.VER_TDS80) {
```

Comment:
```suggestion
        if (tdsVersion >= TDS.VER_TDS80) {
```
Since TDS.VER_TDS80 is our new lowest version, can we simplify to the above? TDS versions listed below:

```
    static final int VER_TDS80 = 0x8000000; // TDS 8.0
    static final int VER_DENALI = 0x74000004; // TDS 7.4
    static final int VER_KATMAI = 0x730B0003; // TDS 7.3B(includes null bit compression)
    static final int VER_YUKON = 0x72090002; // TDS 7.2
    static final int VER_UNKNOWN = 0x00000000; // Unknown/uninitialized
```

Reply:
Ugh. I just noticed there are other places we are checking tdsVersion >= CONSTANT so the new TDS80 version int breaks all that logic. More changes are needed.

Reply:
Also, it's not that simple. Older TDS versions from server responses not listed could look like these:
```
            // Server responds:
            // 0x07000000 -> 7.0         // Notice server response format is different for bwd compat
            // 0x07010000 -> 2000 RTM     // Notice server response format is different for bwd compat
            // 0x71000001 -> 2000 SP1
            // 0x72xx0002 -> 2005 RTM
```
The JDBC driver just got lucky that its range of supported TDS versions happened to align nicely over the past 2 decades. It's much more [complex in SqlClient, for example](https://github.com/dotnet/SqlClient/blob/v4.1.0/src/Microsoft.Data.SqlClient/netfx/src/Microsoft/Data/SqlClient/TdsParser.cs#L4009).

I don't think I want to refactor the logic significantly at this point in time.

Reply:
I see okay, thanks!